composer.json

```diff
@@ -19,7 +19,8 @@
     },
     "require-dev": {
         "codeception/codeception": "^5.0",
-        "phpstan/phpstan": "^1.10.0",
+        "phpstan/phpstan": "1.10.66",
+        "phpstan/phpdoc-parser": "1.25.0",
```

Comment:
why is it added?

Reply:
version incompatibility. CI is failed. so it's fix for Rector

Reply:
should not be composer.lock updated?

Reply:
no it shouldn't. it blocks versions in the require section. Core packages should have lock files

Reply:
>Core packages should have lock files

yes, but should NOT have.